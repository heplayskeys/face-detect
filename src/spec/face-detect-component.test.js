import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import FaceDetect from '../components/face-detect/face-detect.component';

describe('<FaceDetect />', () => {
	it('returns correct face detect snapshot without props', () => {
		const wrapper = shallow(<FaceDetect />);
		expect(shallowToJson(wrapper)).toMatchSnapshot();
	});

	it('returns correct face detect snapshot with props', () => {
		const imageText = 'face-detect-img';
		const wrapper = shallow(<FaceDetect imageURL={imageText} />);
		expect(shallowToJson(wrapper)).toMatchSnapshot();
	});
});
