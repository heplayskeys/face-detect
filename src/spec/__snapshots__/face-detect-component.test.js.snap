// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FaceDetect /> returns correct face detect snapshot with props 1`] = `
<div
  className="ma center"
>
  <div
    className="absolute mt4 db display-image-container"
  >
    <img
      alt="face-detect"
      className="ba br3 b--white shadow-3 animate__animated animate__bounceIn detect-img"
      id="face-detect-img"
      src="face-detect-img"
    />
  </div>
</div>
`;

exports[`<FaceDetect /> returns correct face detect snapshot without props 1`] = `
<div
  className="f4 mt4 dark-red animate__animated animate__bounceIn dib"
/>
`;
